libdnf/dnf-context.cpp

```diff
@@ -330,7 +350,7 @@ dnf_context_get_repo_dir(DnfContext *context)
  *
  * Returns: the base architecture, e.g. "x86_64"
  *
- * Since: 0.1.0
+ * Since: 0.26.1
```

Comment:
I think you might have accidentally updated a version that shouldn't be updated?

Reply:
Oh, yes. I think I was looking at the wrong function...